integration_tests/e2e/make-a-referral.cy.ts

```diff
@@ -1,20 +1,241 @@
+      expect(el.get(0).checkVisibility()).to.eq(false)
+    })
+    cy.findByText(/what type of information to include/i).click()
+    cy.findByText(/include any reasons given/i).then(el => {
+      expect(el.get(0).checkVisibility()).to.eq(true)
+    })
+    cy.findByRole('textbox', { name: /what reasons have been given for the incident/i }).type(
+      'incident reasons foobar123',
+    )
+    checkAxeAccessibility()
+    cy.findByRole('button', { name: /continue/i }).click()
+
+    cy.url().should('include', '/contributory-factors')
+    cy.findByRole('heading', { name: /what are the contributory factors\?/i }).should('be.visible')
+    cy.findByRole('checkbox', { name: /factor1/i }).click()
+    cy.findByRole('checkbox', { name: /factor3/i }).click()
+    checkAxeAccessibility()
+    cy.findByRole('button', { name: /continue/i }).click()
+
+    cy.url().should('include', '/contributory-factors-comments')
+    cy.findAllByRole('link', { name: /add comment/i }).should('have.length', 2)
+    cy.findAllByRole('link', { name: /add comment/i })
+      .first()
+      .should('be.visible')
+      .click()
+    cy.url().should('include', '/code1-comment')
+    cy.findByRole('heading', { name: /add a comment on factor1 factors \(optional\)\?/i }).should('be.visible')
+    cy.findByRole('link', { name: /^back/i }).click()
+    cy.url().should('include', '/contributory-factors-comments')
+    cy.findAllByRole('link', { name: /add comment/i })
+      .first()
+      .should('be.visible')
+      .click()
+    cy.url().should('include', '/code1-comment')
+    cy.findByRole('textbox', { name: /add a comment on factor1 factors \(optional\)\?/i }).type('factor comment')
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.url().should('include', '/contributory-factors-comments')
+    cy.findByRole('link', { name: /edit comment/i }).should('be.visible')
+    checkAxeAccessibility()
+    cy.findByRole('button', { name: /continue/i }).click()
+
+    cy.url().should('include', '/safer-custody')
+    cy.findByRole('heading', { name: /is the safer custody team already aware of this referral\?/i }).should(
+      'be.visible',
+    )
+    cy.findByRole('radio', { name: /yes/i }).click()
+    checkAxeAccessibility()
+    cy.findByRole('button', { name: /continue/i }).click()
+
+    cy.url().should('include', '/additional-information')
+    cy.findByRole('heading', { name: /add additional information \(optional\)/i }).should('be.visible')
+    cy.findByRole('textbox', { name: /add additional information \(optional\)/i }).type('additional info')
+    checkAxeAccessibility()
+    cy.findByRole('button', { name: /continue/i }).click()
+
+    // TODO: We need to check the answers given are correct on CYA page, but it doesnt exist yet
+
+    // We use this for now as CYA page isn't implemented - change this to click back link when it exists
+    cy.url().should('include', '/check-answers')
+    cy.go('back')
+
```

Comment:
the /check-answers page doesn't have a Back link.
once you get there, you're locked into CYA page: most of the other pages should just get you back to the CYA page.

/additional-information would be the final page where you can Back all the way to the first screen.

Reply:
That's why I've used cy.go('back') which is the same as pressing the browser back button and what the comment above it is trying to explain :)

Reply:
what i'm doing in my PR is that, once you've hit the check-answers page, the back url on all the previous pages will be replaced with 'check-answers' (with the exception of the multi-screen sub journey of first 2 screens). so you won't be able to test the back link.

Reply:
Ah I see ok, I'll change it then